src/Service.Host/client/src/containers/parts/PartTemplates.js

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux';
+import { initialiseOnMount } from '@linn-it/linn-form-components-library';
+import partTemplatesActions from '../../actions/partTemplatesActions';
+import partTemplatesSelectors from '../../selectors/partTemplatesSelectors';
+import partTemplateSelectors from '../../selectors/partTemplateSelectors';
+import PartTemplatesSearch from '../../components/parts/PartTemplatesSearch';
+import { getPrivileges } from '../../selectors/userSelectors';
```

Comment:
you won't need to pass privileges into any of your components if you use the resources links I wouldn't think